ocaml/ocaml-bos/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup ocaml     1.1
+
+name                ocaml-bos
+version             0.2.1
+revision            0
+categories          ocaml devel
+maintainers         {pguyot @pguyot} openmaintainer
+license             ISC
+platforms           darwin
+
+description         Basic OS interaction for OCaml
+long_description    Bos provides support for basic and robust interaction with \
+                    the operating system in OCaml. It has functions to access \
+                    the process environment, parse command line arguments, \
+                    interact with the file system and run command line programs.
+
+homepage            https://erratique.ch/software/bos
+master_sites        https://erratique.ch/software/bos/releases/
+
```

Comment:
@pguyot Could you please add `use_bzip2 yes`? Like in https://github.com/macports/macports-ports/pull/23600
That will fix extraction.

Reply:
> That will fix extraction.

…on OS X 10.8 and earlier.